src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.csproj

```diff
@@ -357,4 +358,4 @@
     </EmbeddedResource>
   </ItemGroup>
```

Comment:
It doesn't look like the new test recordings are added here.  You'll need to add these explicitly, with 'Copy If Newer' or 'Copy Always' property